crates/nu-utils/src/sample_config/default_env.nu

```diff
@@ -69,17 +69,13 @@ $env.ENV_CONVERSIONS = {
 }
 
 # Directories to search for scripts when calling source or use
-#
-# By default, <nushell-config-dir>/scripts is added
 $env.NU_LIB_DIRS = [
-    ($nu.default-config-dir | path join 'scripts')
+    # ($nu.default-config-dir | path join 'scripts') # add <nushell-config-dir>/scripts
 ]
 
 # Directories to search for plugin binaries when calling register
-#
-# By default, <nushell-config-dir>/plugins is added
 $env.NU_PLUGIN_DIRS = [
-    ($nu.default-config-dir | path join 'plugins')
+    # ($nu.default-config-dir | path join 'plugins') # add <nushell-config-dir>/plugins
```

Comment:
i'm not sure if people use this or not, but since it's been enabled as default as so long, i think we should either keep it uncommented, or add it to the rust code as a default.

Reply:
i'd make the same answer as the `NU_LIB_DIRS` thread above :wink:

Reply:
We now have 2 issues related to this change. I think we should reverse it because `nu -c` loads the default_env.nu and now we've broken it. https://github.com/nushell/nushell/issues/9863 and https://github.com/nushell/nushell/issues/9833